project/Commands.scala

```diff
@@ -0,0 +1,28 @@
+import sbt.Keys._
+import sbt._
+
+object Commands {
+
+  implicit class CommandOps(project: Project) {
+    def addRunCommand(description: String, javaExtras: Seq[String] = Seq.empty): Project = {
+      import complete.DefaultParsers._
+
+      lazy val runTask = inputKey[Unit](description)
+
+      fork in runTask := false
```

Comment:
Could this be a parameter defaulting to `false`, so we could switch it at will in the main build-file?